README.md

```diff
@@ -1 +1,68 @@
-# Portfolio-App
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio Application
+
+> This is a Portfolio Application built with HTML and CSS
+
+
+## Built With
+
+- HTML
+- CSS
+
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/willy836)
+- Twitter: [@twitterhandle](https://twitter.com/Kimiri836)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/wilson-kimiri-420396235/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] It would be nice to have a customized README file for each project. Please change the information like removing the example information in the getting started and adding clear steps for the user, write your Github, Twitter, and Linkedin names(I see that the links are correct), so this README can look more professional and describe this project.